File: profiles/go/template/src/types/boundaries.go
// Package types defines boundary types for module interfaces.
// 外部输入（HTTP body、CLI args、env vars）必须在入口处用这些类型校验。
package types

type APIResponse struct {
	Status  int                    `json:"status"`
	Message string                 `json:"message"`
	Data    map[string]interface{} `json:"data,omitempty"`
}

File: profiles/python/template/src/types/boundaries.py
"""Boundary types — 模块边界的数据形状定义。
外部输入（HTTP body、CLI args、env vars）必须在入口处用这些类型校验。"""

from dataclasses import dataclass


@dataclass
class APIResponse:
    status: int
    message: str
    data: dict | None = None
